### DISCUSSION TITLE
Using this firmware for E3V2 with ArcWelder

### DISCUSSION THREAD

**Discussion by Gil80** [Jyers/Marlin#1059] [Q&A] [answered]
I know all Marlin 2.x support Arc commands.
This question is just to validate if this fork supports it and if the correct configuration for it in Ocotoprint is to enable this:
![image](https://user-images.githubusercontent.com/8454919/123305083-0ce8e400-d563-11eb-92ac-f8f7accd4448.png)

Thanks for a great firmware!

**Comment by Cal-LF** [comment, marked as answer]
Reference #943

**Reply by Gil80** [reply]
thank you
